docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+
+services:
+
+  consul-server-1:
+    image: consul:latest
+    networks:
+      - trento
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+      - "8600:8600"
+      - "8600:8600/udp"
+    command: "agent -server -bootstrap-expect 3 -ui -client 0.0.0.0"
```

Comment:
In my personal execution I use the `-bind` parameter, but I'm not sure if it is needed.

Reply:
I think it's not needed in this case because the container only has one network interface.

Reply:
Yes, I'm guessing it's binding on `lo` and the port redirection is taking care of allowing it to be reachable outside the container. Regardless, that part has been tested and works so I wouldn't add anything here.